azerothcore/azerothcore-wotlk
# Set pools for Ghostlands Mining

Sets pools for Ghostlands mining, each node has a 60% chance of copper, 30% chance of tin and 10% chance of silver.

**Changes proposed:**

-  Sets pools for mining nodes in the Ghostlands (currently every node spawns a Copper, Tin and Silver on top of each other.
-  


**Target branch(es):**

1.x/2.x etc.


**Issues addressed:**


Closes # https://github.com/azerothcore/azerothcore-wotlk/issues/781


**Tests performed:**
Did you test in-game? Yes
What did you test? Leveled a Blood Elf through the ghostlands to test, didn't spot any bad nodes.
Did you do all these tests on Linux, Mac or Windows? Windows



**How to test the changes:**
Which commands to use? Which NPC to teleport to? Teleport to the Ghostland then check mining nodes.




**Known issues and TODO list:**
